### DISCUSSION TITLE
Temperature not shown?

### DISCUSSION THREAD

**Discussion by boeboe26** [DanielStormApps/Fanny#92] [General]
Why is the CPU/ GPU temperature not shown?

<img width="308" alt="Bildschirmfoto 2024-08-15 um 14 45 55" src="https://github.com/user-attachments/assets/b76ad5c6-6410-4335-a713-b4d935746afd">
<img width="168" alt="Bildschirmfoto 2024-08-15 um 14 45 37" src="https://github.com/user-attachments/assets/bc3df523-6629-4c9d-b780-ff293c9ebabb">

**Comment by sedaray** [comment]
Not shown on my MacPro Mojave as well.  If I change to anything besides "Average" in preferences, it shows readings of - -ºF.
